vms/pom/locators/manageShiftPageLocators.py

```diff
@@ -1,8 +1,12 @@
 class ManageShiftPageLocators(object):
 
     CANCEL_SHIFT_PATH = '//table//tbody//tr[1]//td[5]'
+    CLEAR_SHIFT_HOURS_TEXT = 'html/body/div[2]/form/div/div[1]/h3'
```

Comment:
we should `//` if we are using any index after a tag.

Reply:
I just moved it, didn't write it. I'll update it.

Reply:
okay.